### DISCUSSION TITLE
Signing of a Flet App[Question]

### DISCUSSION THREAD

**Discussion by meistermaxb04** [flet-dev/flet#3220] [Q&A] [answered]
### Question

Hi,
I'm working on a Flet App for my Company.
For publishing we need to use `flet pack` because `flet build` can't pack the app to a onefile .exe as far as I am informed.
We also need to sign the App through the windows signtool to ensure our Users can only open the Flet App we created.
The Problem:
When opening the Flet App, there is a Flet.exe that is created in the tmp-folder that is opened and not signed. 
So I don't have the possibility to sign the App, because it creates executables that are not signed and therefore cannot be opened.

Is there a solution?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment, marked as answer]
Right, `flet build` cannot pack into a single file, but it creates a standalone app, without a wrapper (as PyInstaller does) that can be signed, etc.

You can do a self-extracting archive or MSI if you want to deliver a single file to customers. Anyway, all those single-file solutions is just an executable with app archive inside.
